vivisect/qt/funcgraph.py

```diff
@@ -83,16 +84,29 @@ def contextMenuEvent(self, event):
         else:
             menu = QtGui.QMenu(parent=self)
 
-        viewmenu = menu.addMenu('view   ')
-        viewmenu.addAction("Save frame to HTML", ACT(self._menuSaveToHtml))
-        #TODO: Refresh menu item :)
+        self.viewmenu = menu.addMenu('view   ')
+        self.viewmenu.addAction("Save frame to HTML", ACT(self._menuSaveToHtml))
+        self.viewmenu.addAction("Refresh", ACT(self.refresh))
```

Comment:
any particular reason you're assigning this as an instance local? i dont see it re-used anywhere ( and if we *are* going to keep "viewmenu" as an instance local, then it should be declared and initialized to None in the ctor. :)